docs/features/notifications.md

```diff
@@ -20,6 +20,9 @@ We support the following set of notifications along with badge / taskbar count o
 ## macOS
 macOS notifications are native chrome notifications which appear like any other desktop app notifications on a Mac.
 Emojis are supported in the toast.
```

Comment:
This is not the case anymore. We support both custom & native notifications for both Windows & macOS.

@KiranNiranjan Can you update the documentation accordingly?